sippy-ng/src/releases/ReleaseOverview.js

```diff
@@ -28,12 +28,10 @@ import VariantCards from '../jobs/VariantCards'
 
 export const REGRESSED_TOOLTIP =
   'Shows the most regressed items this week vs. last week, for those with more than 10 runs, excluding never-stable and techpreview.'
-export const NOBUG_TOOLTIP =
-  'Shows the list of tests ordered by least successful and without a bug, for those with more than 10 runs'
-export const TRT_TOOLTIP =
-  'Shows a curated list of tests selected by the TRT team'
 export const TWODAY_WARNING =
   'Shows the last 2 days compared to the last 7 days, sorted by most regressed, excluding never-stable and techpreview.'
+export const TOP_FAILERS_TOOLTIP =
+  'Shows the list of tests ordered by least working percentage.'
```

Comment:
least working = failing? I like the use of working meaning pass + flake, but for least working I am thinking "failing" is clearer. Let me know if that sounds wrong though.

Reply:
Make sense, I used "failing"